### PR TITLE
[REF] extract add median to stats

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Code clean up preliminary to resolving performance issues

Before
----------------------------------------
Code hard to read

After
----------------------------------------
Code more readable

output ugly but unchanged
screenshot 2019-02-17 17 37 30

Technical Details
----------------------------------------
Partial of https://github.com/civicrm/civicrm-core/pull/13607 - adding 'has-tests' which were merged in #13605

Comments
----------------------------------------

### CODE REVIEW

File: CRM/Contact/BAO/Query.php [hunk 1]
```diff
@@ -6688,4 +6682,22 @@ protected function addModeToStats(&$summary, $from, $where) {
...
+   * multiple
+   *
+   * see https://lab.civicrm.org/dev/core/issues/720
+   *
+   * @param array $summary
+   * @param string $where
+   * @param string $from
+   */
+  protected function addMedianToStats(&$summary, $where, $from) {
+    $medianSQL = "{$from} {$where} ND civicrm_contribution.contribution_status_id = 1 ";
```
Comment (colemanw) [inline CRM/Contact/BAO/Query.php]: `ND`?
Copy-paste error?
I'm interested to see if the tests catch it. If so then adding `merge-on-pass` is a no-brainer.

Comment (eileenmcnaughton) [reply CRM/Contact/BAO/Query.php]: opps - I re-tweaked that commit when pulling it out from the borg....  yeah tests will fail

**Comment by eileenmcnaughton** [comment]
@colemanw voila - proof

**Comment by colemanw** [comment]
Cool

**Comment by eileenmcnaughton** [comment]
ok fixed now....

**Pull request merged**